C/manual/ch_Finance-Quote.xml

```diff
@@ -330,25 +345,62 @@ Failed to initialize Finance::Quote: missing_modules Finance::Quote JSON::Parse
 </screen>
       </informalexample>
 
-      <para>In this case, &app-fq; is not installed correctly and therefore cannot be used for course retrieval with &app;.
-        Please install &app-fq; according to the instructions at
+      <para>In this case, &app-fq; is not installed correctly and therefore cannot be used for course retrieval
+        with &app;. Please install &app-fq; according to the instructions at
         <xref linkend="fq-install" />.
       </para>
     </sect2>
 
     <sect2 id="fq-print-quotes">
       <title>Displaying Quotes in a Terminal Window</title>
 
+      <abstract>
+        <para>The <replaceable>--quotes dump</replaceable> mode provides quotes for a source and a list of of
+          symbols in a format that is easy for humans to read. This is useful to verify that a
+          particular online course source is accessible and provides data.
+        </para>
+
+        <para>You can also check to see if the symbol you want to use for your online
+          price retrieval is available at the desired price source is
+          known.
+        </para>
+      </abstract>
+
+      <tip>
+        <para>With the use of <userinput>gnucash-cli --quotes dump</userinput>.
+          <footnoteref linkend="fn_gnc-fq-old" /> you can test symbols faster than from within
+          &app;, in case an error occurs during the retrieval with &app;. The output of the script
+          is better suited for analysis, because not all working symbols will be are retrieved as
+          well, if <guibutton>Get Quotes</guibutton> in the <xref linkend="tool-price" />.
+        </para>
+      </tip>
+
       <para>To display a quote for one or more stocks or the exchange rate for one or more currencies you can
-        use <userinput>gnucash-cli --quotes dump</userinput> as follows. It offers two output forms
-        for non-currency securities and one for currency exchange rates.
+        use <userinput>gnucash-cli --quotes dump</userinput> as follows:
+        <cmdsynopsis>
+          <command>gnucash-cli</command>
+          <arg choice="opt">-V</arg>
+          <arg choice="req">--quotes</arg>
```

Comment:
Don't mix long and short options, it's confusing. Either `--verbose` and `--quotes` or `-V` and `-Q`.

Reply:
@gjanssens Why must the short options be uppercase? I knew that only for inversed options.

Reply:
In `<cmdsynopsis>` you should use `<group …>`:
```
          <command>gnucash-cli</command>
          <group choice="opt">
            <arg choice="plain">-V</arg>
            <arg choice="plain">--verbose</arg>
          </group>
          <group choice="req">
            <arg choice="plain">-Q</arg>
            <arg choice="plain">--quotes</arg>
          </group>
          <arg choice="req">dump</arg>
         [–]
```

Reply:
After some playing I believe you want something like:
```
        <cmdsynopsis>
          <command>gnucash-cli</command>
          <arg choice="req">
            <group choice="req">
              <arg choice="plain">-Q</arg>
              <arg choice="plain">--quotes</arg>
            </group>
            dump
            <group choice="opt">
              <arg choice="plain">-V</arg>
              <arg choice="plain">--verbose</arg>
            </group>
            <replaceable>Source</replaceable>
            <arg choice="plain" rep="repeat"><replaceable>Symbol</replaceable></arg>
          </arg>
        </cmdsynopsis>
```
Unfortunatly yelp renders it one arg per line while html puts it all in one line.
I reordered the args because -V has only a meaning in the dump context.

Reply:
@fellen :
I don't think we need the braces for Source and Symbol, as `chgrp --help` shows: `Usage: chgrp [OPTION]... GROUP FILE...`